OracleGoldenGate/21c/README.md

```diff
@@ -66,8 +66,9 @@ $ docker run \
     -e OGG_ADMIN=<admin user name> \
     -e OGG_ADMIN_PWD=<admin password> \
     -e OGG_DEPLOYMENT=<deployment name> \
-    -e RUN_COMMAND=<script or command> \
-    -e RUN_COMMAND_BACKGROUND=<true | false> \
+    -e SETUP_USER_SCRIPTS=<path to setup scripts> \
```

Comment:
The convention established in the `OracleDatabase` project is a pre-coded location of either:
- `"$ORACLE_BASE"/scripts/setup`
- `"$ORACLE_BASE"/scripts/startup`

I would expect something similar here, with:
- `"$OGG_HOME"/scripts/setup`
- `"$OGG_HOME"/scripts/startup`

Is there really any value in allowing control over the script directory location when a volume mount already controls the content of those directories?

Reply:
This issue needs resolution.

Reply:
The issue has been resolved now. Not allowing any more control over the scripts directory as the volume mount controls it.